feat: Display exchange logo in Vera download form

PanchoBubble opened the pull request:
The Vera mobile download component now receives the exchange data as a prop. This allows the component to display the exchange logo within the email input form, specifically beside the "Use your email ID associated with" text. This improves clarity for the user by indicating which exchange their email should be associated with when requesting the download link.

CjS77 commented on _updates/2020-11-25-update-42.md:
```diff
@@ -0,0 +1,46 @@
+
+## Major changes since v0.6.0
+
+- [#2431](https://github.com/tari-project/tari/pull/2431) Database refactor to remove checkpoints, fix failure on reorg and more efficient calculation of MMR roots
```
```suggestion
- [#2431](https://github.com/tari-project/tari/pull/2431) Database refactor to replace MMR checkpoints with a more efficient and simpler approach; fix reorg failures
```

CjS77 commented on _updates/2020-11-25-update-42.md:
```diff
@@ -0,0 +1,46 @@
+- [#2444](https://github.com/tari-project/tari/pull/2444) DHT connectivity waits for comms connectivity before starting
+- [#2427](https://github.com/tari-project/tari/pull/2427) Update merge mining runtime, README, Win install
+- [#2420](https://github.com/tari-project/tari/pull/2420) Show base node chain tip and sync status in the console wallet
+- [#2419](https://github.com/tari-project/tari/pull/2419) Add optional fee_per_gram field to console wallet Send Tx form
```
Does this allow variable fees in console wallet? If so, you can promote it more as a "new feature"

CjS77 commented on _updates/2020-11-25-update-42.md:
```diff
@@ -0,0 +1,46 @@
+- [#2421](https://github.com/tari-project/tari/pull/2421) Fix QR code rendering in the console wallet
+- [#2423](https://github.com/tari-project/tari/pull/2423) Plumb in the balance in the console wallet
+- [#2415](https://github.com/tari-project/tari/pull/2415) Prevent loop in peer sync by storing all peers attempted
+- [#2383](https://github.com/tari-project/tari/pull/2383) Implement daemon-mode in tari_base_node
```
Bump to major and add colour

CjS77 reviewed (commented):
Nice. Do you mind making a few suggested edits to add some wight to the post? Otherwise looks great.

sdbondi reviewed (approved):
Looks good - all I can think of suggesting is touting that there are minor security improvements (specifically in #2377)

StriderDM commented:
LGTM

PanchoBubble committed:
feat: Display exchange logo in Vera download form

The Vera mobile download component now receives the exchange data as a prop. This allows the component to display the exchange logo within the email input form, specifically beside the "Use your email ID associated with" text. This improves clarity for the user by indicating which exchange their email should be associated with when requesting the download link.
